CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/DocumentRequestHandler.swift

```diff
@@ -20,21 +20,12 @@ public class DocumentRequestHandler {
         //////////////
         case "document_create":
             let id: String? = (args.get(name: "id"))
-            let dictionary: [String: Any]? = (args.get(name: "dictionary"))
+            let dictionary: [String: Any] = (args.get(name: "dictionary")) ?? [String: Any]()
 
             if id != nil {
-                if dictionary == nil {
-                    return MutableDocument(id: id)
-                } else {
-                    return MutableDocument(id: id, data: dictionary)
-                }
+                return MutableDocument(id: id, data: dictionary)
```

Comment:
you can use only the `MutableDocument(id:, data: )` constructor here as well, since arg `id ` is nullable. if its empty, it will create a random UUID as `id`.

Reply:
ok

Reply:
https://docs.couchbase.com/mobile/2.8.0/couchbase-lite-swift/Classes/MutableDocument.html#/s:18CouchbaseLiteSwift15MutableDocumentC2id4dataACSSSg_SDySSypGSgtcfc